src/legacy/core_plugins/kibana/public/home/kibana_services.js

```diff
@@ -18,9 +18,23 @@
  */
 
 import { uiModules } from 'ui/modules';
+import { npStart } from 'ui/new_platform';
+import { createUiStatsReporter, METRIC_TYPE } from '../../../ui_metric/public';
+import { TelemetryOptInProvider } from './telemetry_opt_in';
 
 export let indexPatternService;
+export let showTelemetryOptIn;
```

Comment:
I mentioned it before, but I'd really like to see a better boolean naming for this one. I realize it's mostly existing code, but this doesn't seem a very typical way to for a service to expose a value.